omod/src/main/java/org/openmrs/module/appui/UiSessionContext.java

```diff
@@ -46,6 +48,33 @@ public UiSessionContext() {
 
     }
 
+    public String getViewProvidersPrivilege()
+    {
+    	PrivilegeConstants privilegeConstants = new PrivilegeConstants();
+    	String viewProvidersPrivilege = "";
+    	try {
+    		// openmrsVersion 1.9
+    		Class<?> privilegeConstantsClass = privilegeConstants.getClass();
+    		Field viewProvidersPrivilegeConstant = privilegeConstantsClass.getDeclaredField("VIEW_PROVIDERS");
+    		viewProvidersPrivilege = (String)viewProvidersPrivilegeConstant.get(privilegeConstants);
+    	} catch (NoSuchFieldException ex) {
+    		// do nothing
+ 		} catch (IllegalAccessException ex) {
+			// do nothing
+		}
+    	try {
+    		// openmrsVersion 2.0
+    		Class<?> privilegeConstantsClass = privilegeConstants.getClass();
+    		Field viewProvidersPrivilegeConstant = privilegeConstantsClass.getDeclaredField("GET_PROVIDERS");
+    		viewProvidersPrivilege = (String)viewProvidersPrivilegeConstant.get(privilegeConstants);
+    	} catch (NoSuchFieldException ex) {
+    		// do nothing
+ 		} catch (IllegalAccessException ex) {
+			// do nothing
+		}
+    	return viewProvidersPrivilege;
+    }
+    
```

Comment:
The privilege constant has changed from VIEW_PRIVILEGES in earlier openmrs version to GET_PRIVILEGES in 2.0.